pygmt/tests/test_legend.py

```diff
@@ -0,0 +1,98 @@
+"""
+Tests for legend
+"""
+import pytest
+
+from .. import Figure
+from ..helpers import GMTTempFile
```

Comment:
```suggestion
from ..exceptions import GMTInvalidInput
from ..helpers import GMTTempFile
```

Needed for the new test below.